### ISSUE TITLE
network_mode: host vs. port bindings

### ISSUE THREAD

**Issue by maddhin** [volker76/tuya_mqtt.net#10]
Hi,

afaik, the use of network_mode: host together with ports: / port bindings has been depreciated. Or actually, the port bindings has been ignored if host was used. 

Afaik tuya_mqtt.net uses port 80 to publish itself, or? Would it be possible to change this when using host to 8889? 

Sorry if this already works, I cannot test as my image is not running. But at least the docker-compose docu should be updated;) 

Cheers!

**Comment by volker76** [comment]
you are right network_mode:host and port bindings does not go together. To have the internal web-gui exposed to another port inside the container, use the environment variable -e ASPNETCORE_URLS=http://*:8889

see
[Run-in-docker-standalone](https://github.com/volker76/tuya_mqtt.net/wiki/Run-in-docker-standalone)

**Comment by volker76** [comment]
@maddhin I run docker-compose in the documented port settings under 20.10.22-qnap7 and docker desktop 4.22. do you get woith your docker any deprication warning?

**Comment by maddhin** [comment]
on Raspbian 11 32bit w/ latest docker and docker-compose there is a warning message that one or the other should be used. If i remember correct the warning comes with docker -up
